docs/getting-started.md

```
@@ -0,0 +1,100 @@
+## General
+
+To use the refarch-templates, you need to have Docker installed on your
+machine. Inside the `stack` folder, you will find a `docker-compose.yml`
+file that will spin up everything needed for local development. 
+
+When developing locally, the stack should always be running. 
+### Download Templates
+
+You can download the templates or clone them directly from the GitHub
+repository.
+```
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)